src/www/ui/api/Controllers/GroupController.php

```diff
@@ -165,4 +167,59 @@ public function deleteGroupMember($request, $response, $args)
 
     return $response->withJson($returnVal->getArray(), $returnVal->getCode());
   }
+
+  /**
+   * Get users with their roles from a given group
+   *
+   * @param ServerRequestInterface $request
+   * @param ResponseHelper $response
+   * @param array $args
+   * @return ResponseHelper
+   */
+  public function getGroupMembers($request, $response, $args)
+  {
+    $userId = $this->restHelper->getUserId();
+    /** @var UserDao */
+    $userDao = $this->restHelper->getUserDao();
+    $groupMap = $userDao->getAdminGroupMap($userId, $_SESSION[Auth::USER_LEVEL]);
+    $res = null;
+
+    if (empty($groupMap)) {
+      $res = new Info(400, "You have no permission to manage any group.", InfoType::ERROR);
+    } else {
+
+      /**
+       * Get the group id from the params *
+       **/
+      $groupId = intval($args['id']);
+
+      /**
+       * The query to get the list of users with corresponding roles from the group. *
+       **/
+
+      $dbManager = $this->dbHelper->getDbManager();
+
+      $stmt = __METHOD__ . "getUsersWithGroup";
+      $dbManager->prepare($stmt, "select user_pk, user_name, user_status, user_desc, group_user_member_pk, group_perm
+         FROM users INNER JOIN group_user_member gum ON gum.user_fk=users.user_pk AND gum.group_fk=$1
+         ORDER BY user_name");
```

Comment:
Since only `user_pk` and `group_perm` as used in the code following, no need to fetch it from DB.
```suggestion
      $dbManager->prepare($stmt, "SELECT user_pk, group_perm
         FROM users INNER JOIN group_user_member gum ON gum.user_fk=users.user_pk AND gum.group_fk=$1;");
```

Reply:
Reminder @dushimsam